### PR TITLE
Test indentation script on Mac OSX

### PR DESCRIPTION
**Pull request by masterleinad**
Just for testing purposes.

### CODE REVIEW

**Comment by tjhei** [comment]
How about merging this? This way we know when things break in the future, right?

**Comment by masterleinad** [comment]
Fine with me. Let me see if I can get it working.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -52,12 +54,14 @@ jobs:
     - name: configure
       run: |
         CC=mpicc CXX=mpic++ cmake -D CMAKE_BUILD_TYPE=Debug -D DEAL_II_CXX_FLAGS='-Werror' -D DEAL_II_WITH_MPI=on .
+        ./contrib/utilities/download_clang_format
```
Comment (tjhei) [inline .github/workflows/main.yml]: would you mind moving this up into ``setup``?

**Comment by masterleinad** [comment]
Should work now.

**Comment by tjhei** [comment]
This is okay with me as is. The run scripts run with ``pipefail``, so a failure should be detected.

**Comment by masterleinad** [comment]
> The run scripts run with `pipefail`, so a failure should be detected.

Yes, I checked.

**Pull request merged**